mon-pix/app/components/challenge-statement.js

```diff
@@ -46,6 +46,18 @@ export default class ChallengeStatement extends Component {
     return 'challenge_statement_' + this.args.challenge.id;
   }
 
+  @action
+  readTheInstruction() {
+    if (!window.speechSynthesis.speaking) {
+      const utterance = new SpeechSynthesisUtterance();
+
+      utterance.text = this._removeMarkDown(this.challengeInstruction);
```

Comment:
Coté pix1d, on a essayé de mettre en place l'oralisation en suivant ce qui a été fait dans cette PR. Mais on s'est demandé pourquoi pas prendre le texte déjà rendu dans la page plutôt que la version markdown avec les caractères spéciaux, qui nécessite un nettoyage. Donc, on a plutôt opté pour cette solution
```suggestion
      utterance.text = document.getElementsByClassName('challenge-statement-instruction__text')[0].innerText;
```

Reply:
j'ai jamais trop aimé parser le html pour récupérer le text qui m'intéresse. mais ça marche oui 😅 .  Là tu dépend de la classe. si un changement de code advient. la lecture ne marcherait plus :(

Reply:
Alors oui effectivement, on devient dépendant de la classe css c'est le coté négatif, mais le coté pratique c'est que tu n'as plus à gérer les différents caractères spéciaux liés au markdown, et on ne se prend plus la tête sur ce qu'on doit garder ou non dans le texte.
Après, à voir s'il y a une manière moins dépendante de la classe css.